dora/core/server/master/src/test/java/alluxio/master/predicate/DatePredicateTest.java

```diff
@@ -0,0 +1,49 @@
+package alluxio.master.predicate;
+
+import alluxio.proto.journal.Job.FileFilter;
+import alluxio.wire.FileInfo;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Unit tests for {@link DatePredicate}.
+ */
+public class DatePredicateTest {
+  /**
+   * The exception expected to be thrown.
+   */
+  @Rule
+  public final ExpectedException mThrown = ExpectedException.none();
+
+  @Test
+  public void testDatePredicateFactories() {
+    FileFilter filter = FileFilter.newBuilder().setName("lastModifiedDate").setValue("2020/01/01").build();
```

Comment:
one question is that if we have the wrong date format like (2020-01-01) what's the behavior? It's always good to have unit test covering the exceptional case